### DISCUSSION TITLE
Can't get RaspAP to work as a normal WAP using a wired connection

### DISCUSSION THREAD

**Discussion by MarkBinner** [RaspAP/raspap-webgui#1647] [General]
I installed RaspAP on a Pi4 with 4GB of RAM. The LAN port is connected to my router with a cable. I tried to set up the Pi in bridged mode so that the IP addresses come from the DHCP server in the router. The ethernet port is set up in the router with a DHCP reservation of 192.168.0.40 but it never gets that IP address. Using Angry IP scanner I got addresses of 192.168.0.18 for one MAC address and 192.168.0.19 for the other. What I want is for eth0 to be assigned its reserved address and the wlan0 to be a free IP address in the same range. I just want to use the pi as an access point in places where wi-fi coverage to patchy but I intend to always have it cabled to the router. Can I do that or I am seriously misunderstanding how it works?
Thanks for any help/advice?
Mark B

**Comment by billz** [comment]
Start here https://docs.raspap.com/bridged/#troubleshooting

**Reply by MarkBinner** [reply]
Bill,
Thanks for the link but I had read that and didn't see anything there that helped. Did I miss the relevant bit?
Mark B

**Reply by billz** [reply]
The vast majority of bridged AP problems stem from issues related to upstream DHCP servers. If your Pi isn't able to obtain an IP from your upstream router, for whatever reason, start with the default (non-bridged) AP mode. This should suffice for what you're trying to achieve.
